pkg/intcode/intcode_test.go

```diff
@@ -18,14 +18,19 @@ func TestComputeNoInput(t *testing.T) {
 	}
 
 	for expectedRes, inst := range computeInstructions {
-		test.EqualInt(t, Compute(inst, Multiply), expectedRes)
+		outputs := make(chan int, 1)
+		Compute(inst, Multiply, make(chan int), outputs)
+		test.EqualInt(t, <-outputs, expectedRes)
 	}
 }
 
 func TestComputeInput(t *testing.T) {
 	const input = 42
 
-	test.EqualInt(t, Compute([]int{3, 0, 4, 0, 99}, Output, input), input)
+	inputs, outputs := make(chan int, 1), make(chan int, 1)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [avoid magic numbers like '1', create a named constant for it](https://app.codacy.com/manual/afarbos/aoc/pullRequest?prid=5498712)